src/protocol.h

```diff
@@ -276,6 +276,8 @@ enum ServiceFlags : uint64_t {
     // collisions and other cases where nodes may be advertising a service they
     // do not actually support. Other service bits should be allocated via the
     // BIP process.
+
+    NODE_REPLACE_BY_FEE = (1 << 26),
```

Comment:
I don't think this makes sense if the entire network (or even most of it) will be expected to enable full RBF.

Reply:
(Although note that this PR does NOT currently enable full RBF by default...)

Reply:
Correct. I wanted something as minimal as possible, such that nodes could opt-out(of honoring non-rbf).

Perhaps this flag is over-kill, but :man_shrugging: